build.sbt

```diff
@@ -235,7 +241,7 @@ val argonautV = "6.2.1"
 val circeV = "0.11.2"
 val circeJava8V = "0.11.1"
 val jwtCirceV = "4.0.0"
-val jacksonV = "2.9.2"
+val jacksonV = "2.11.3"
```

Comment:
there's already 2.11.4 but why not use 2.12.x ?

Reply:
2.11.3 was the largest of all the transitive dependencies. The library is not used in the NK code.